src/features/user/TreeMapper/components/InterventionPage.tsx

```diff
@@ -35,14 +35,10 @@ const ImageSliderSingle = dynamic(
 
 interface Props {
   setSelectedLocation: SetState<
-    SamplePlantLocation | PlantLocationMulti | PlantLocationSingle | null
+    SampleIntervention | InterventionMulti | InterventionSingle | null
   >;
-  location:
-    | PlantLocationMulti
-    | PlantLocationSingle
-    | SamplePlantLocation
-    | null;
-  plantLocations?: PlantLocation[];
+  location: InterventionMulti | InterventionSingle | SampleIntervention | null;
+  intervention?: Intervention[];
 }
```

Comment:
1. The use of the term `location` is unclear, particularly after renaming 'plantLocation` to `intervention` in this file and across the app.
2. `intervention?: Intervention[]` - pluralize the variable name as it represents an array, not a single interventions.